src/build-info/src/lib.rs

```diff
@@ -39,7 +39,11 @@ pub const DUMMY_BUILD_INFO: BuildInfo = BuildInfo {
 impl BuildInfo {
     /// Constructs a human-readable version string.
     pub fn human_version(&self) -> String {
-        format!("v{} ({})", self.version, &self.sha[..9])
+        if self == &DUMMY_BUILD_INFO {
```

Comment:
Whoop, this seems like treating the symptom but not the cause. I think we should just come up with a better `DUMMY_BUILD_INFO`. Maybe stick "v0.0.0+dummy" in the `version` field and `0000000000000000000000000000000000000000` in the `she` field?